src/ihex2cmd/ihex2cmd.cpp

```diff
@@ -4,9 +4,9 @@
 
 using namespace std;
 
-#define CHECK(condition, errno) \
+#define TRS_CHECK(condition, errno) \
```

Comment:
Could you please change the prefix from "TRS_" to "CMD_"?

Reply:
done